SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/GameModeReplicationTest/GameModeReplicationTest.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "GameModeReplicationTest.h"
+
+#include "Net/UnrealNetwork.h"
+
+#include "GameFramework/GameStateBase.h"
+
+AGameModeReplicationTestGameMode::AGameModeReplicationTestGameMode()
+{
+	NetCullDistanceSquared = 0;
+}
+
+void AGameModeReplicationTestGameMode::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	DOREPLIFETIME(AGameModeReplicationTestGameMode, ReplicatedValue);
+}
+
+bool AGameModeReplicationTestGameMode::IsNetRelevantFor(const AActor* RealViewer, const AActor* ViewTarget,
+														const FVector& SrcLocation) const
+{
+	return false;
+}
+
+/**
+ * This test checks that GameModes:
+ * * Only owned by a single worker
+ * * Replicate between servers
+ * * Don't replicate to clients
+ */
+
+AGameModeReplicationTest::AGameModeReplicationTest()
+{
+	Author = TEXT("Dmitrii");
+	Description = TEXT("Test GameMode replication");
+
+	bReplicates = true;
+	bAlwaysRelevant = true;
+}
+
+void AGameModeReplicationTest::MarkWorkerGameModeAuthority_Implementation(int WorkerId, bool bHasGameModeAuthority)
```

Comment:
Doesn't look like `WorkerId` is used for anything.

Reply:
Yeah, in the end this didn't get used. Will remove.